portfolio/src/main/webapp/style.css

```diff
@@ -66,11 +101,33 @@ h3 {
   width: 50%;
 }
 
-p {
+p{
   color: #242d49;
   font-family: 'Montserrat', sans-serif;
 }
 
+.font-style{
```

Comment:
[4.2.5](https://google.github.io/styleguide/htmlcssguide.html#Property_Name_Stops): Use a space between the last selector and the declaration block.